kvm.go

```diff
@@ -259,7 +274,11 @@ func (d *Driver) validatePrivateNetwork() error {
 		"255.255.255.0",
 		"192.168.42.2",
 		"192.168.42.254")
-	network, err = d.conn.NetworkDefineXML(xml)
+
+	if err != nil {
+		return err
+	}
```

Comment:
Something seems wrong here.  If the lookup fails on line 228, you set up the XML definition on 272, but then just error out instead of actually creating the network.  We should try to create the network if it doesn't exist.